install_files/php/boot.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-define('OCTOBER_MINIMUM_PHP_VERSION_ID', 70000);
+define('OCTOBER_MINIMUM_PHP_VERSION_ID', 70800);
```

Comment:
@gergo85 The number above would need to be `70008` to represent `7.0.8`.
```suggestion
define('OCTOBER_MINIMUM_PHP_VERSION_ID', 70008);
```